RZViewActions/UIView+RZViewActions.m

```diff
@@ -128,6 +142,17 @@ - (void)_runWithCompletion:(RZViewActionCompletion)completion
 
 @end
 
+#pragma mark - RZViewSpringAction
+
+@implementation RZViewSpringAction
+
+- (void)_runWithCompletion:(RZViewActionCompletion)completion
```

Comment:
Underscore-prefixed methods are reserved by Apple.
![justin is wrong](https://cloud.githubusercontent.com/assets/464574/5780635/2d94f52c-9d7a-11e4-9235-9743b98d4c3b.png)